packages/vkui/src/components/Group/Group.module.css

```
@@ -53,6 +50,19 @@
   }
 }
 
+.Group--sizeX-compact {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+@media (--sizeX-compact) {
+  .Group--sizeX-none {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.Group--mode-none.Group--sizeX-regular,
```

```suggestion
.Group--sizeX-regular.Group--mode-none,
```
чтобы одинаковый стиль был

---

✅ fix